src/TwigComponent/src/Test/InteractsWithTwigComponents.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Test;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+trait InteractsWithTwigComponents
```

Comment:
I am just wondering why do we use trait here? In my use case, I gonna create a dedicated test class for my component, and I not gonna mixed my components tests with other tests

Reply:
This is just a pattern I've found useful with my other testing libraries. I like using traits to avoid forcing a single _type_ of test.

```php
class MyComponentTest extends KernelTestCase
{
    use InteractsWithTwigComponents, ResetDatabase, Factories;

    // ...
}
```

I get the above would still work if we made this trait into and abstract `TwigComponentTestCase`. I'm just trying to be consistent: _Make a standard kernel test, and inject the test features you need via traits._

Reply:
Ho yes using this trait with foundry just rocks! Thanks for all of this 😁